src/tests/ftest/util/server_utils.py

```diff
@@ -373,6 +373,7 @@ def clean_files(self, verbose=True):
             verbose (bool, optional): display clean commands. Defaults to True.
         """
         clean_cmds = []
+        NUM_IO_SERVERS = 2
```

Comment:
The `DaosServerYamlParameters` class has a `servers_per_host` parameter which can be accessed through `num_io_servers = self.self.get_config_value("servers_per_host")`.  You would also want to use a lower-case variable name here when following PEP-8 standards.  The `servers_per_host` value also drives the size of the `self.manager.job.yaml.server_params` list: https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/server_utils_params.py#L145-L148.  Since this list is already looped through on line 377, the `for srv_idx in range(NUM_IO_SERVERS):` appears to be redundant resulting in the same `sudo ipcrm -M <key>` command appearing in the `clean_cmds` list.  So ultimately this line is not needed:
```suggestion
```